stable/rook-ceph/values.yaml

```diff
@@ -0,0 +1,294 @@
+  #      limits:
+  #        memory: 256Mi
+  #        cpu: 100m
+  # Set provisonerTolerations and provisionerNodeAffinity for provisioner pod.
+  # The CSI provisioner would be best to start on the same nodes as other ceph daemons.
+  # provisionerTolerations:
+  #    - key: key
+  #      operator: Exists
+  #      effect: NoSchedule
+  # provisionerNodeAffinity: key1=value1,value2; key2=value3
+  provisionerNodeAffinity: node-role.rook-ceph/cluster=any,mon,mgr,mon-mgr,osd,mon-mgr-osd
+  # Set pluginTolerations and pluginNodeAffinity for plugin daemonset pods.
+  # The CSI plugins need to be started on all the nodes where the clients need to mount the storage.
+  # pluginTolerations:
+  #    - key: key
+  #      operator: Exists
+  #      effect: NoSchedule
+  # pluginNodeAffinity: key1=value1,value2; key2=value3
+  # pluginNodeAffinity: node-role.rook-ceph/client=true
+  #cephfsGrpcMetricsPort: 9091
+  #cephfsLivenessMetricsPort: 9081
+  #rbdGrpcMetricsPort: 9090
+  # Enable Ceph Kernel clients on kernel < 4.17. If your kernel does not support quotas for CephFS
+  # you may want to disable this setting. However, this will cause an issue during upgrades
+  # with the FUSE client. See the upgrade guide: https://rook.io/docs/rook/v1.2/ceph-upgrade.html
+  forceCephFSKernelClient: true
+  #rbdLivenessMetricsPort: 9080
+  #kubeletDirPath: /var/lib/kubelet
+  #cephcsi:
+    #image: registry.suse.com/ses/7/cephcsi/cephcsi:3.0.0.0
+  #registrar:
+    #image: registry.suse.com/ses/7/cephcsi/csi-node-driver-registrar:v1.2.0
+  #provisioner:
+    #image: registry.suse.com/ses/7/cephcsi/csi-provisioner:v1.6.0
+  #snapshotter:
+    #image: registry.suse.com/ses/7/cephcsi/csi-snapshotter:v2.1.1
+  #attacher:
+    #image: registry.suse.com/ses/7/cephcsi/csi-attacher:v2.1.0
+  #resizer:
+    #image: registry.suse.com/ses/7/cephcsi/csi-resizer:v0.4.0
+
+enableFlexDriver: false
+enableDiscoveryDaemon: true
+
+## if true, run rook operator on the host network
+# useOperatorHostNetwork: true
+
+## Rook Discover configuration
+## tolerations: Array of tolerations in YAML format which will be added to discover deployment
+## nodeAffinity: Set to labels of the node to match
+discover:
+#   tolerations:
+#   - key: key
+#     operator: Exists
+#     effect: NoSchedule
+#   nodeAffinity: key1=value1,value2; key2=value3
+  nodeAffinity: node-role.rook-ceph/cluster=any,mon,mgr,mon-mgr,osd,mon-mgr-osd
+
+# In some situations SELinux relabelling breaks (times out) on large filesystems, and doesn't work with cephfs ReadWriteMany volumes (last relabel wins).
+# Disable it here if you have similar issues.
+# For more details see https://github.com/rook/rook/issues/2417
+enableSelinuxRelabeling: true
+
+# Writing to the hostPath is required for the Ceph mon and osd pods. Given the restricted permissions in OpenShift with SELinux,
+# the pod must be running privileged in order to write to the hostPath volume, this must be set to true then.
+hostpathRequiresPrivileged: false
```

Comment:
shouldn't this be true as well for caasp?

Reply:
We have not seen this to be a problem in CaaSP environments.